engineyard/ey-cookbooks-stable-v5
# [YT-CC-1218] New Relic Server Monitoring agent should no longer be installed

## Description of your patch

This patch removes New Relic Server Monitoring package instllation, because it have been replaced by New Relic Infrastructure.

## Recommended Release Notes

None

## Estimated risk

Low

## Components involved

New Relic cookbook

## Description of testing done

See QA instructions - tests were done on a solo environment.

## QA Instructions

1. Create a simple PHP application (git://github.com/engineyard/howto.git) can be used for this testing.
2. Create environment using QA stack created from this PR branch.
  * Environment type - Single Instance.
  * Application Server Stack - PHP-FPM.
3. Enable New Relic Addon for the created environment. Apply changes
4. Run command `eix sysmond`. Ensure `sys-apps/newrelic-sysmond` package is not listed.
